search.go

```diff
@@ -34,3 +35,24 @@ type NilSearch bool
 func (ns NilSearch) Search(p SearchParams) ([]Result, error) {
 	return nil, ErrSearchNotSupported
 }
+
+// MockSearch wraps a registry.Datasets and is used for testing purposes only
```

Comment:
"MockSearch is a very naive implementation of search that wrapt registry.Datasets and only checks for exact substring matches of dataset's `meta.title` property. It's mainly intended for testing purposes."